frontend/src/Context/UserContext.ts

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { createContext } from "react";
+
+export type UserContextType = {
+  user: any;
+  setUser: any;
+}
```

Comment:
It would be better to have types instead of any

export type UserContextType = {
  user: User | null;
  setUser: (user: User | null) => void;
}